src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -191,6 +207,8 @@ public new static RevitDynamoModel Start(IRevitStartConfiguration configuration)
         private RevitDynamoModel(IRevitStartConfiguration configuration) :
             base(configuration)
         {
+            DisposeLogic.IsShuttingDown = false;
```

Comment:
Not necessary, by default it is `false`. We need to reset `IsShuttingdown` here because it is the only place to reset `IsShuttingDown` back.